test/test_prepare_fieldmap.py

```diff
@@ -28,12 +28,22 @@ def setup(self):
 
     def test_prepare_fieldmap_1_echo(self):
```

Comment:
Similar answer to my comment above, I specified prelude in the test below because I'm testing for prelude specifically in that test (it's the only test using prelude in `test_prepare_fieldmap`